### ISSUE TITLE
Delete this?

### ISSUE THREAD

**Issue by job13er** [ciena-attic/ember-frost-blueprints#2]
Do we still want/need this repo? I believe we were gonna just put blueprints in `ember-frost-core` now, right?

**Comment by sglanzer-fire** [comment]
Yeah, I think we can at a minimum archive this, but likely a full delete is fine too

**Comment by ghost** [comment]
Folks, who is admin for the ciena-attic org?
